doc/generators.yaml

```diff
@@ -81,17 +81,17 @@ data:
     data:
       "1": stdout.py 1 # prints `1` to stdout, which is piped to 1.in
 
-      # To indicate a manual testcase already present in the corresponding directory, specify it with an empty value.
+      # To indicate a unlisted testcase already present in the corresponding directory, specify it with an empty value.
       # Prefer specifying a path to a .in file as in case `3` below.
       # E.g., this indicates that `data/sample/2.in` (and optionally `data/sample/2.ans`) are existing non-generated files.
       "2":
-      # The copy: key indicates a manual testcase that will be copied
+      # The copy: key indicates a unlisted testcase that will be copied
```

Comment:
Shouldn't this still be `manual`? :thinking: Same for `manual_cases` below, and in `skel/generators/generators.yamlL45`

Reply:
thanks for checking :) it was inevitable this would end up too greedy on the search-and-replace